packages/batch/src/BatchProcessor.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Process native partial responses from SQS, Kinesis Data Streams, and DynamoDB
+ */
+import {
+  BasePartialBatchProcessor,
+  BaseRecord,
+  FailureResponse,
+  SuccessResponse,
+} from '.';
+
+class BatchProcessor extends BasePartialBatchProcessor {
+  /**
+   * Process a record with instance's handler
+   * @param record Batch record to be processed
+   * @returns response of success or failure
+   */
+  public async processRecord(
+    record: BaseRecord
+  ): Promise<SuccessResponse | FailureResponse> {
+    try {
+      const data = this.toBatchType(record, this.eventType);
+      const result = await this.handler(data);
+
+      return this.successHandler(record, result);
+    } catch (e) {
```

Comment:
```suggestion
    } catch (error) {
```

Reply:
Please consider applying thsi for all occurrences of `catch (e)`.

We discourage using single-char variables to encourage using meaningful names for all variables.

Even though in this case it's pretty clear that `e` stands for error, depending on the proficiency/level/experience of the reader this meaning could get lost. Writing the full name disambiguates what `e` is.